src/utils/__tests__/csvColumnAnalysis.test.ts

```
@@ -260,6 +303,12 @@ describe('csvColumnAnalysis', () => {
     });
 
     it('handles CSV files with quoted headers', async () => {
+      const csvContent =
+        '"Project Title","Project Status","Company Name"\n"Test","completed","TestCo"';
+      const file = new File([csvContent], 'test.csv', { type: 'text/csv' });
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)